x-pack/plugin/sql/src/main/java/org/elasticsearch/xpack/sql/expression/gen/script/Param.java

```diff
@@ -22,6 +22,6 @@ T value() {
 
     @Override
     public String toString() {
-        return String.format(Locale.ROOT, "{%s=%s}", prefix(), value);
+        return LoggerMessageFormat.format(null, "{{}={}}", prefix(), value);
```

Comment:
Why use the `null` prefix -and not use alternative method.

Reply:
@costin I use the one with the prefix and using `null` because of signature mixup:
`String, String` calls the one with the prefix and not the `String, Object...` one...